overview/default.nix

```diff
@@ -163,7 +163,7 @@ let
           prefixLength = 2;
         in
         optionalString (!empty projectOptions) ''
-          ${heading 2 "options" "Options"}
+          ${heading 2 "service" "Options"}
```

Comment:
Aren't we showing programs yet? Where do those go?

Reply:
Oh good point, they are actually rendered together and should be split out into a different heading.

Reply:
But I suppose this should be addressed in a different PR?

Reply:
https://github.com/ngi-nix/ngipkgs/issues/1006